### DISCUSSION TITLE
How to implement the setting of the cursor moving quickly according to the word

### DISCUSSION THREAD

**Discussion by magiceses** [kingToolbox/WindTerm#2268] [General]
![image](https://github.com/kingToolbox/WindTerm/assets/50326171/bd6cc263-a3fb-4ba0-a4e7-58a0e762c884)

under macos, option + left and right arrow keys, keep outputting letters on the screen, so how to change it to hold down option + left, to jump between --amend, commit, and git words?

Anybody help?

**Comment by ali-hejazi** [comment]
Same problem
